test/runtests.jl

```diff
@@ -5,6 +5,38 @@ const GROUP = get(ENV, "GROUP", "All")
 const is_APPVEYOR = Sys.iswindows() && haskey(ENV,"APPVEYOR")
 const is_TRAVIS = haskey(ENV,"TRAVIS")
 
+using LibGit2
+r = LibGit2.GitRepo("..")
+commit = LibGit2.peel(LibGit2.GitCommit, LibGit2.head(r))
+msg = LibGit2.message(commit)
+
+@show msg[1:6]
+@show msg[7:18]
+@show msg[7:12]
+@show msg[15:18]
+
+if length(msg) >= 5 && msg[1:5] == "base:"
```

Comment:
It's generally not a good idea to base indexing of a String on `length`. As mentioned in [the docs](https://docs.julialang.org/en/v1/manual/strings/index.html#String-Basics-1)
> `lastindex(str)` is not in general the same as `length(str)` for a string, because some Unicode characters can occupy multiple "code units".

In such cases [regular indexing might fail](https://docs.julialang.org/en/v1/manual/strings/index.html#Unicode-and-UTF-8-1).

In general, I guess `split` and `startswith` might be helpful for what you want to check.